modules/seqexec/model/src/main/scala/seqexec/model/enum/BatchCommandState.scala

```diff
@@ -4,12 +4,13 @@
 package seqexec.model.enum
 
 import cats.Eq
+import seqexec.model.ClientID
 
 sealed trait BatchCommandState extends Product with Serializable
 
 object BatchCommandState {
   case object Idle extends BatchCommandState
-  case object Run extends BatchCommandState
+  final case class Run(clientId: ClientID) extends BatchCommandState
```

Comment:
`clientId` was needed to start the sequences.

Reply:
this change is breaking the build as boopickle serialization is broken